src/app/ecosystem-explorer/project-form/components/EcosystemProjectForm.tsx

```diff
@@ -169,15 +170,9 @@ export function EcosystemProjectForm({
               For best quality, please submit a white logo with a transparent
               background, at least 1000px by 1000px, and under 100KB. You can
               use tools like{' '}
-              <a
-                href="https://squoosh.app/"
-                className="text-brand-300 underline hover:text-brand-400"
-                rel="noopener noreferrer"
-                target="_blank"
```

Comment:
It would nice to keep `target="_blank"` here. If you click on the link then come back, the form state resets.

Reply:
And I suppose `rel="noopener noreferrer"` too

Reply:
Yes! Good catch.